apps/web/src/components/Profile/index.tsx

```diff
@@ -114,7 +114,7 @@ const ViewProfile: NextPage = () => {
       />
       <GridLayout className="pt-6">
         <GridItemFour>
-          <Details profile={profile as any} following={!!following} setFollowing={setFollowing} />
+          <Details profile={profile as Profile} following={!!following} setFollowing={setFollowing} />
```

Comment:
why !! is used here you are converting number to boolean?

Reply:
It's not number its `boolean | null`

Reply:
Also true `!!` is a good way, I changed to `Boolean(following)`